docs/documentation/upgrading/topics/changes/changes-26_2_0.adoc

```diff
@@ -19,6 +19,17 @@ the `X-Forwarded-Port` header with the desired port.
 The required JAR for the Oracle JDBC driver that needs to be explicitly added to the distribution has changed.
 Instead of providing `ojdbc11` JAR, use `ojdbc17` JAR as stated in the https://www.keycloak.org/server/db#_installing_the_oracle_database_driver[Installing the Oracle Database driver] guide.
 
+=== H2 Credentials
+
+With this version, the default H2 based `dev-file` database changed its credentials. While migrating from an instance using this dev only database is not supported, you may be able to continue to use your existing H2 database if you explicitly provide the old defaults for the database username and password. For example in the `keycloak.conf` specify:
+
+[example]
+====
+db-username=sa
+
```

Comment:
Shall we add an example for containers?

Reply:
Doesn't seem necessary as this isn't a supported case, and the user should be able to extrapolate how to configure as needed.

Reply:
Sounds good to me! Thanks for confirming!